scripts/update/Updater.php

```diff
@@ -49,5 +51,112 @@ public function update($initialVersion)
         }
 
         $this->skip('1.1.0', '1.4.2');
+
+        if ($this->isVersion('1.4.2')) {
+
+            OntologyUpdater::syncModels();
+
+            $serviceManager = $this->getServiceManager();
+
+            $deliveryContainerService = new DeliveryContainerService();
+            $deliveryContainerService->setServiceManager($serviceManager);
+
+            $serviceManager->register(DeliveryContainerService::SERVICE_ID, $deliveryContainerService);
+
+            $plugins = [
+                'probes' => [
```

Comment:
could you please ensure this works great with env/customer that already have their `probes` plugin.